state-manager-frontend/src/app/page.tsx

```
@@ -0,0 +1,504 @@
+      setCurrentStep(stepIndex);
+
+      switch (step.id) {
+        case 'register-nodes':
+          const registerResponse = await apiService.registerNodes(
+            namespace,
+            {
+              runtime_name: runtimeName,
+              nodes: [sampleNode]
+            },
+            apiKey
+          );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)